packages/lib-subject-viewers/.npmignore

```diff
@@ -0,0 +1,13 @@
+.cache
```

Comment:
This is copy pasted from lib-react-components and outdated. Please delete until ready to publish to npm, or revise specifically for this library.